sonar-dotnet-shared-library/src/main/java/org/sonar/plugins/dotnet/tests/WildcardPatternFileProvider.java

```diff
@@ -43,16 +43,14 @@
   private static final String ANY_PATTERN = "?";
 
   private final File baseDir;
-  private final String directorySeparator;
 
-  public WildcardPatternFileProvider(File baseDir, String directorySeparator) {
+  public WildcardPatternFileProvider(File baseDir) {
     this.baseDir = baseDir;
-    this.directorySeparator = directorySeparator;
   }
 
   Set<File> listFiles(String pattern) {
 
-    List<String> elements = Arrays.asList(pattern.split(Pattern.quote(directorySeparator)));
+    List<String> elements = Arrays.asList(pattern.split("["+ Pattern.quote(File.separator) +"/]"));
```

Comment:
can the `"["+ Pattern.quote(File.separator) +"/]"` be in a static final field? to avoid doing string concatenations at each call?